src/components/NewTask.jsx

```diff
@@ -13,7 +13,9 @@ import { AuthContext } from "../services/providers/authProvider";
  * in which the user can create a new task by filling put the fields and clicking the "add task" button
  * This component also presists the task on the firebase database
  */
-export default function NewTask() {
+export default function NewTask(props) {
```

Comment:
Just nitpicking here but I think I saw in other files their styling is as follows: 

NewTask( { onNewTask} ) { 
... 
}

Reply:
Hmmmm, don't worry - I see that there are other places that they use props as well